From c7d683e9ca0b10fddf166374c62ed3dbfd437d85 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 18 Mar 2018 09:35:10 +0100
Subject: [PATCH 1/1] configs: sunxi: enable BLK, DM_MMC, DM_SCSI

doc/driver-model/MIGRATION.txt requires to move block drivers to the
the driver model by v2018.05

So make the switch for the SUNXI block drivers.

Select BLK, DM_MMC, and DM_SCSI.

DM_USB dpends on USB but there is no necessity to make it depend on
DISTRO_DEFAULTS.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 arch/arm/Kconfig | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index 2c52ff025a..b608f01beb 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -715,13 +715,16 @@ config ARCH_SUNXI
 	select BINMAN
 	select CMD_GPIO
 	select CMD_MMC if MMC
-	select CMD_USB if DISTRO_DEFAULTS
+	select CMD_USB if USB
 	select DM
+	select BLK
 	select DM_ETH
 	select DM_GPIO
 	select DM_KEYBOARD
+	select DM_MMC
+	select DM_SCSI
 	select DM_SERIAL
-	select DM_USB if DISTRO_DEFAULTS
+	select DM_USB if USB
 	select OF_BOARD_SETUP
 	select OF_CONTROL
 	select OF_SEPARATE
-- 
2.16.2

